docs/caa.md

```diff
@@ -0,0 +1,107 @@
+list](https://mailarchive.ietf.org/arch/msg/spasm/vHL260X6Zb2C0VSwDwa8VZC1Kw0),
+we achieved consensus that tree-climbing in CAA is not ideal, and there's [an
+erratum](https://www.rfc-editor.org/errata/eid5065) for the CAA RFC removing it.
+That erratum still needs to be voted in by the CA/Browser Forum by September 8
+for it to be official.
+
+# CAA errors
+
+Since Let's Encrypt checks CAA records before every certificate we issue, sometimes
+we get errors even for domains that haven't set any CAA records. When we
+get an error, there's no way to tell whether we are allowed to issue for the
+affected domain, since there could be CAA records present that forbid issuance,
+but are not visible because of the error.
+
+If you receive CAA-related errors, try a few more times against our [staging
+environment](https://letsencrypt.org/docs/staging-environment/) to see if they
+are temporary or permanent. If they are permanent, you will need to file a
+support issue with your DNS provider, or switch providers. If you're not sure
+who your DNS provider is, ask your hosting provider.
+
+Some DNS providers that are unfamiliar with CAA initially reply to problem
+reports with "We do not support CAA records." Your DNS provider does not need
+to specifically support CAA records; it only needs to reply with a
+NOERROR response for unknown query types (including CAA). Returning other
+opcodes, including NOTIMP, for unrecognized qtypes is a violation of [RFC
+1035](https://tools.ietf.org/html/rfc1035).
+
+# SERVFAIL
+
+One of the most common errors that people encounter is SERVFAIL. Most often this
+indicates a failure of DNSSEC validation. If you get a SERVFAIL error, your
+first step should be to use a DNSSEC debugger like
+[dnsviz.net](http://dnsviz.net/). If that doesn't work, it's possible that your
+nameservers generate incorrect signatures only when the response is empty. And
+CAA responses are most commonly empty.  For instance, PowerDNS [had this bug in
+version 4.0.3 and below](https://community.letsencrypt.org/t/caa-servfail-changes/38298/2?u=jsha).
+
+If you don't have DNSSEC enabled and get a SERVFAIL, the second most likely
+reason is that your authoritative nameserver returned NOTIMP, which as described
+above is an RFC 1035 violation; it should instead return NOERROR with an empty
+response. If this is the case, file a bug or a support ticket with your DNS provider.
+
+Lastly, SERVFAILs may be caused by outages at your authoritative nameservers.
+Check the NS records for your nameservers and ensure that each server is
+available.
+
+# Timeout
+
+Sometimes CAA queries timeout. That is, the authoritative name server never
+replies with an answer at all, even after multiple retries. Most commonly this
+happens when your nameserver has a misconfigured firewall in front of it that
+drops DNS queries with unknown qtypes. File a support ticket with your DNS
+provider and ask them if they have such a firewall configured.
```

Comment:
I tried to figure out a good way to work in "we really mean it," but wound up just saying "and will need to be fixed by your DNS provider." We can strengthen later based on actual conversations.